### ISSUE TITLE
Cannot download the self distribution tar from BinTray 

### ISSUE THREAD

**Issue by liu-xinhui** [moquette-io/moquette#681]
<img width="1054" alt="image" src="https://user-images.githubusercontent.com/7698511/174696039-c795ec26-3b32-4fb2-a2b8-7eb2c73a02a8.png">

The download you’re looking for isn’t available any longer. Your download was hosted on JFrog Bintray, which has been deprecated. Please contact the project owner to get your downloads.

**Comment by hylkevds** [comment]
Good point, we need to change the documentation to point to maven central:
https://repo1.maven.org/maven2/io/moquette/

**Comment by alcandelario** [comment]
Any plans to update the readme? The maven.org link also does not appear to have the moquette-0.15.tar.gz file referenced in the BinTray URL

**Comment by andsel** [comment]
Starting from version 0.18 the tar.gz is shipped in the GH release, checkout https://github.com/moquette-io/moquette/releases/tag/v0.18.0
